ansible/tasks/security/main.yml

```diff
@@ -1,90 +1,101 @@
 ---
-- name: Restrict tempfile namespaces
+- name: Configure pam_limits(8)
+  copy:
+    dest: /etc/security/limits.conf
+    content: |
+      1000:65535           soft    nproc           150
+      1000:65535           hard    nproc           200
+      1000:65535           soft    nofile          4096
+      1000:65535           hard    nofile          10240
+
+- name: Install script for enforcing per-user ressource limits
+  copy:
+    dest: /etc/security/limits.sh
+    mode: "a+x"
+    content: |
+      #!/bin/sh -e
+
+      PAM_UID=$(getent passwd "${PAM_USER}" | cut -d: -f3)
+
+      if [ "${PAM_UID}" -ge 1000 ]; then
+          /bin/systemctl set-property "user-${PAM_UID}.slice" \
+                         CPUQuota=50% MemoryLimit=512M BlockIOWeight=10
+      fi
+
+- name: "pam_namespace:  Disable services with known escapes"
```

Comment:
This isn't really pam_namespace related.

Reply:
Fair, that should just be `pam:`